care/facility/migrations/0390_alter_dailyround_consciousness_level.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.2.5 on 2023-10-11 05:02
+
+from django.db import migrations, models
+
+
+def migrate_consciousness_level(apps, schema_editor):
+    daily_round_model = apps.get_model("facility", "DailyRound")
+    records = daily_round_model.objects.all()
+
+    entries = []
+
+    for record in records:
+        consciousness_level = record.consciousness_level
+        if consciousness_level == 25:
+            record.consciousness_level = 0
+            entries.append(record)
+    daily_round_model.objects.bulk_update(entries, ["consciousness_level"])
```

Comment:
Consider optimizing the query

```suggestion
    qs = daily_round_model.objects.filter(consciousness_level=25)
    qs.update(consciousness_level=10)
```